README.md

```diff
@@ -49,7 +49,7 @@ class TopicsController < ApplicationController
 end
 ```
 
-This method will act as a `before_action` that triggers when spam is detected (honeypot field has some value). By default, it responds with no content (only headers: `head(200)`). This is a good default, since the bot will surely read the response code and will think that it has achieved to submit the form properly. But, anyway, you can define your own callback by passing a method to the `on_spam` option:
+This method will act as a `before_action` that triggers when spam is detected (honeypot field has some value). By default, it responds with no content (only headers: `head(200)`). This is a good default, since the bot will surely read the response code and will think that it has achieved to submit the form properly. But, anyway, you can define your own callback by passing a method to the `on_spam` option. n.b. that you should _not_ name your method `on_spam`, as this will collide with an internal method of the same name.
```

Comment:
What about moving this note just below the code example (where you added the deleted comment) 🙏🏼? IMHO it would be clearer 👌🏼

Reply:
done